lib/txt/news

```diff
@@ -4,31 +4,18 @@
 07-04-17 : The world command now reports the correct number of distinct
```

Comment:
Why did this file suddenly change so much?

Reply:
cleaned up date formatting, out of order entries, indentation, and trailing whitespace.

Reply:
honestly I can't believe it still applies - we need more news entries for player-visible changes